includes/class-notices.php

```diff
@@ -498,22 +537,41 @@ static function print_frontend_notice( $message, $echo = true, $notice_type = 'w
 			return '';
 		}
 
+		/**
+		 * Change auto_dismissible to dismissible and set the value to true
+		 *
+		 * @since 1.8.14
+		 */
 		$default_notice_args = array(
-			'auto_dismissible' => false,
+			'dismissible' => true,
 			'dismiss_interval' => 5000,
 		);
 
 		$notice_args = wp_parse_args( $notice_args, $default_notice_args );
 
+		/**
+		 * Added to give support to backward compatibility to auto_dismissible.
+		 * Check if auto_dismissible is set and it true then unset and change dismissible parameter value to auto
+		 *
+		 * @since 1.8.14
+		 */
+		if ( isset( $notice_args['auto_dismissible'] ) ) {
```

Comment:
@raftaar1191 I can see this code repeat multiple times. We can create a function which we handle deprecated functions.

Reply:
Yes we can do that